scripts/core/ui/components/select2.tsx

```diff
@@ -86,112 +89,166 @@ export class Select2<T> extends React.Component<IProps<T>, IState> {
         this.search = throttle(searchFn, 300, {leading: false});
     }
 
+    componentDidMount() {
+        // capture all scroll events and close autocomplete on scroll
+        // unless that scroll event is coming from the autocomplete itself
+        window.addEventListener('scroll', (e) => {
```

Comment:
can't find where is this listener removed

Reply:
forgot it :upside_down_face: